.profile.d/heroku-metrics-daemon.sh

```diff
@@ -10,6 +10,24 @@ export HEROKU_METRICS_PROM_PORT=$(expr $PORT + 1)
 export HEROKU_PROM_METRICS_ENDPOINT=${HEROKU_METRICS_PROM_ENDPOINT}
 export HEROKU_PROM_METRICS_PORT=${HEROKU_METRICS_PROM_PORT}
 
+BUILD_DIR=/app
+DOWNLOAD_URL=$(curl --retry 3 -s https://api.github.com/repos/heroku/agentmon/releases/latest | grep "browser_download_url" | awk -F': ' '{print $2}' | sed -e 's/"//g')
+if [ -z "${DOWNLOAD_URL}" ]; then
+    echo "!!!!! Failed to find latest agentmon. Please report this as a bug. Metrics collection will be disabled this run."
```

Comment:
let's prefix this with something so that it can be discerned in the log stream from output coming from the user's app